Feature/smart tv (#26)
Vualto/vudrm-public-docs: docs/source/integrations/smart_tv/tizen.md

# Tizen

Samsung TV's use the Tizen SDK extension for providing applications to their Smart TV platforms.

## Basic setup

Samsung provide extensive documentation showing how to get a development environment up and running, including this [quick start guide](https://developer.samsung.com/tv/develop/getting-started/quick-start-guide).

## DRM support

PlayReady DRM is currently the only DRM supported by Samsung.

## Player Partners support

All of our player partners provide mechanisms to make use of VUDRM, and as Tizen only supports PlayReady, the documentation for each can be followed to implement.  

## Native support

Tizen has an inbuilt player within the SDK which can be used to provide encrypted playback utilising PlayReady DRM.

> Full documentation of the `avplayer` can be found on Samsung's developer [documentation](https://developer.samsung.com/tv/develop/legacy-platform-library/ovw008/index)

Reviewer: ```suggestion
Full documentation of the `avplayer` can be found on Samsung's developer [documentation](https://developer.samsung.com/tv/develop/legacy-platform-library/ovw008/index).
```

> implementing a PlayReady solution can be done with the following code sample.

Reviewer: ```suggestion
Implementing a PlayReady solution can be done with the following code sample.
```

> Full dicumentatuion of Samsungs PlayReady support can be found [Tizen PlayRead documentation](https://developer.samsung.com/tv/develop/legacy-platform-library/tut00064/index)

Reviewer: ```suggestion
For further information on Samsungs PlayReady support please visit the [Tizen PlayReady documentation](https://developer.samsung.com/tv/develop/legacy-platform-library/tut00064/index).
```

Reviewer: Can I at least have the full stop at the end of the sentence?

> Full dicumentatuion of Samsungs PlayReady support can be found [Tizen PlayRead documentation](https://developer.samsung.com/tv/develop/legacy-platform-library/tut00064/index).

Reviewer: ```suggestion
For Samsungs full documentation on PlayReady support please visit the [Tizen PlayRead documentation](https://developer.samsung.com/tv/develop/legacy-platform-library/tut00064/index).
```

```javascript
avplayObj.show();
avplayObj.open(
    url,
    {
        drm : {
            type : "Playready",
            company : 'Microsoft Corporation',
            deviceID : '1'
        }
    }
);
avplayObj.play(
    self.playSuccess,
    function (error) {
        console.error(error.message);
    },
    0
);

let customData = "<YOUR VUDRM TOKEN>"
avplayObj.setPlayerProperty(3, customData, customData.length);

> let la_url ="https://playready-license.drm.technology/rightsmanager.asmx";

Reviewer 2: javascript should be camalCase
